### PR TITLE
MTM-49613 Concepts guide screenshot update.

### PR DESCRIPTION
**Pull request by eniosultan**

### CODE REVIEW

**Comment by BeateRixen** [comment]
@eniosultan The images admin-event-processing and device-info can be removed entirely.

**Review by reey — APPROVED** [review]

**Comment by BeateRixen** [comment]
@eniosultan We discussed to remove the Machine Learning Workbench from the app switcher image as this will no longer be integrated in 10.17.

File: content/concepts/applications-bundle/extensible-device-registration.md [hunk 1]
```diff
@@ -63,9 +63,7 @@ There are two types of extensions:
 
 ### Single device registration
 
-After enabling the `extensibleDeviceRegistration` extension type, the **Devices** > **Register device** menu in the Device Management application is being extended with an entry corresponding to the extension `name` property:
-
-![Select guide](/images/concepts-guide/extensible-device-registration/register-device-menu-with-extensible-device-reg.png)
+After enabling the `extensibleDeviceRegistration` extension type, the **Devices** > **Register device** menu in the Device Management application is being extended with an entry corresponding to the extension `name` property.
```
Comment (BeateRixen) [inline content/concepts/applications-bundle/extensible-device-registration.md:66]:
```suggestion
After enabling the `extensibleDeviceRegistration` extension type, the **Devices** > **Register device** menu in the Device Management application is extended with an entry corresponding to the extension `name` property.
```

File: content/concepts/applications-bundle/extensible-device-registration.md [hunk 2]
```diff
@@ -173,9 +171,7 @@ The following diagram visualizes the single device registration flow:
 
 Many device integrations require the registration of many devices at the same time. Currently, all protocols have to rely on the bulk registration mechanism of the platform, which often either requires too many fields or requires custom fields to be added. The latter ones can however so far not be validated, as the core directly creates devices -- and microservices and agents have no control over the properties being written to the managed objects.
 
-After enabling the `extensibleBulkDeviceRegistration` extension type, the Device management > Devices > Register device  `Bulk device registration` modal is being displayed with an extended wizard entry corresponding to the extension `name` property:
-
-![Select guide](/images/concepts-guide/extensible-device-registration/register-device-menu-with-extensible-device-reg.png)
+After enabling the `extensibleBulkDeviceRegistration` extension type, the Device management > Devices > Register device  `Bulk device registration` modal is being displayed with an extended wizard entry corresponding to the extension `name` property.
```
Comment (BeateRixen) [inline content/concepts/applications-bundle/extensible-device-registration.md:174]:
```suggestion
After enabling the `extensibleBulkDeviceRegistration` extension type, the Device management > Devices > Register device  `Bulk device registration` modal is displayed with an extended wizard entry corresponding to the extension `name` property.
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**